README.md

```diff
@@ -27,7 +27,8 @@ If you run into any problems, chances are your problem has already been solved i
 The notebook is designed to be run in demo mode by default using a pre-trained model in under 5 minutes on any reasonable Deep Learning GPU such as nVidia K80/P40/P100/V100/TitanV.
```

Comment:
Might want to edit it so that it is specific to the Azure VM SKU in which K80/etc maps to

Reply:
added #366 to track since this is out of scope for the PR